test/inventory/group_vars/all.yml

```diff
@@ -6,3 +6,10 @@ openshift_cluster_content:
   - name: Create Projects
     file: "{{ inventory_dir }}/../files/projects/projects.yml"
     file_action: create
+- object: testparamsfromvars
+  content:
+  - name: Create Projects
+    template: "{{ playbook_dir }}/files/secrets/template.yml"
+    params: "{{ playbook_dir }}/params/test-param-file"
```

Comment:
Can't find this file. Have you tested to use together both parameters file and parameters from inventory variables?